docs/version_releases.rst

```diff
@@ -0,0 +1,20 @@
+Version releases
+================
+
+Create new release
+------------------
+
+How create new release you can read `this <https://help.github.com/en/articles/creating-releases>`_.
+
+Requirements
+------------
+- New release should based on master branch.
+- New release tag must have name something like 1.0.0.
```

Comment:
I would say: "New release tag should look like "1.0.0".

Reply:
Changed.